src/Data/Camera/CameraRepository.cs

```diff
@@ -21,6 +22,10 @@ public CameraRepository(Context context)
             .Select(CameraFactory.MakeModel)
             .ToList();
 
+        public IEnumerable<MCamera> ByParameter(PCamera parameter) => _context.Set<ECamera>()
+            .ToList()
+            .Select(CameraFactory.MakeModel);
```

Comment:
Should we also use our parameter to filter the output here?

Reply:
@danidoff Can you help us here, please?

Reply:
Done